doc/DOWNLOADS.md

```diff
@@ -6,6 +6,12 @@ If you want to download the latest unstable build and help test Nicotine+, see [
 
 ## GNU/Linux, *BSD, Solaris
 
+### Flatpak
```

Comment:
Flatpak is not the preferred installation method, so it should definitely not appear at the top.

The priority order is in major distros at the top, starting with Ubuntu/Debian, Arch, etc, then the alternative packing solutions towards the bottom.